src/vm/threadpoolrequest.h

```diff
@@ -184,10 +184,9 @@ class ManagedPerAppDomainTPCount : public IPerAppDomainTPCount {
     ADID m_id;
     TPIndex m_index;
     DECLSPEC_ALIGN(MAX_CACHE_LINE_SIZE) struct {
-        BYTE m_padding1[64 - sizeof(LONG)];
+        BYTE m_padding1[MAX_CACHE_LINE_SIZE - sizeof(LONG)];
         // Only use with VolatileLoad+VolatileStore+FastInterlockCompareExchange
         LONG m_numRequestsPending;
-        BYTE m_padding2[64];
```

Comment:
OK I put all the padding arrays back and added one in ThreadCount to be consistent with UnfairSemaphore